src/components/chat/chat-panel.tsx

```diff
@@ -25,16 +27,21 @@ export const ChatPanel: React.FC<IProps> = ({ activeNavTab, documentKey, onClose
             : undefined;
   }, [document, postCommentMutation]);
   const newCommentCount = unreadComments?.length || 0;
-
   return (
     <div className={`chat-panel ${activeNavTab}`} data-testid="chat-panel">
       <ChatPanelHeader activeNavTab={activeNavTab} newCommentCount={newCommentCount}
                        onCloseChatPanel={onCloseChatPanel} />
-      <CommentCard
-        activeNavTab={activeNavTab}
-        onPostComment={postComment}
-        postedComments={comments}
-      />
+      {((activeNavTab ==="problems") || (activeNavTab ==="teacher-guide") || documentKey)
```

Comment:
Seems like we ought to be able to replace this test with a `focusDocument` test -- if there's a `focusDocument` then we should show the comment card, otherwise not. That makes the code more robust because the rules for when to show the comment card may change, but they should always correspond to the `focusDocument`.